app/routes/htmlRoutes.js

```diff
@@ -0,0 +1,12 @@
+// const path = require("path");
```

Comment:
app folder doesn't exist anymore, Could you please incorporate your changes in the routes folder in the root folder?
Thanks